### PR TITLE
Remove CGL RNA-seq pipeline (resolves #446)

### LINKED ISSUE #446: Remove RNA-seq pipeline from toil-scripts
Once RNA-seq repo has pushed a stable release to PyPi.

### PR DESCRIPTION
**Pull request by jvivian**

### CODE REVIEW

**Comment by jvivian** [comment]
I touched so many files due to a dependency import in rnaseq_cgl that I moved to toil_scripts/__init__.py until BD2KGenomics/toil-lib#29 adds it to toil-lib. 

@hannes-ucsc / @fnothaft — Ready for review.

File: src/toil_scripts/__init__.py [hunk 1]
```diff
@@ -19,3 +21,16 @@ def download_from_s3_url(file_path, url):
         key.get_contents_to_filename(file_path)
     finally:
         s3.close()
+
+
+def generate_file(file_path, generate_func):
```
Comment (fnothaft) [inline src/toil_scripts/__init__.py]: Why don't we move this into toil_lib?

Comment (jvivian) [inline src/toil_scripts/__init__.py]: Addressed that in my first comment

Comment (fnothaft) [inline src/toil_scripts/__init__.py]: Sorry, missed that earlier. Shall we resolve https://github.com/BD2KGenomics/toil-lib/issues/29 upstream, then propagate that change here?

Comment (jvivian) [inline src/toil_scripts/__init__.py]: Ya, I'd prefer to fix it in this PR. I'll go ahead and PR 29 against the release/1.0.x branch. Once that's merged and a new release is on PyPi I'll make the change on this PR.

Comment (fnothaft) [inline src/toil_scripts/__init__.py]: +1

**Comment by jvivian** [comment]
@hannes-ucsc — Ready for review.

**Pull request merged**